src/test/e2e/25_helm_test.go

```diff
@@ -55,7 +55,23 @@ func testHelmChartsExample(t *testing.T) {
 	require.Contains(t, e2e.StripMessageFormatting(stdErr), "chart \"asdf\" version \"6.4.0\" not found")
 	require.Contains(t, e2e.StripMessageFormatting(stdErr), "Available charts and versions from \"https://stefanprodan.github.io/podinfo\":")
 
-	// Create the package (with a registry override to test that as well)
+	// Create a test package (with a registry override (host+subpath to host+subpath) to test that as well)
+	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io/stefanprodan=docker.io/stefanprodan", "--tmpdir", tmpdir, "--confirm")
+	require.NoError(t, err, stdOut, stdErr)
+
+	// Create a test package (with a registry override (host to host+subpath) to test that as well)
+	// expect to fail as ghcr.io is overriden and the expected final image doesn't exist but the override works well based on the error message in the output
+	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io=docker.io/noway", "--tmpdir", tmpdir, "--confirm")
+	require.Error(t, err, stdOut, stdErr)
+	require.Contains(t, string(stdErr), "docker.io/noway/stefanprodan/podinfo:6.4.0")
+
+	// Create a test package (with a registry override (host+subpath to host) to test that as well)
+	// works same as the above failing test
+	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io/stefanprodan=docker.io", "--tmpdir", tmpdir, "--confirm")
+	require.Error(t, err, stdOut, stdErr)
+	require.Contains(t, string(stdErr), "docker.io/podinfo:6.4.0")
```

Comment:
```suggestion
	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io=localhost:555/noway", "--tmpdir", tmpdir, "--confirm")
	require.Error(t, err, stdOut, stdErr)
	require.Contains(t, string(stdErr), "localhost:555/noway/stefanprodan/podinfo:6.4.0")

	// Create a test package (with a registry override (host+subpath to host) to test that as well)
	// works same as the above failing test
	stdOut, stdErr, err = e2e.Zarf("package", "create", "examples/helm-charts", "-o", "build", "--registry-override", "ghcr.io/stefanprodan=localhost:555", "--tmpdir", tmpdir, "--confirm")
	require.Error(t, err, stdOut, stdErr)
	require.Contains(t, string(stdErr), "localhost:555/podinfo:6.4.0")
```

Reply:
Would be better to point these to localhost since theoretically `noway` could be a valid docker org in dockerhub someday (which could make this brittle/_potentially_ insecure) https://hub.docker.com/search?q=noway

Reply:
makes sense, something I will keep in mind
I have updated the PR